docs/topics/web-overview.md

```diff
@@ -0,0 +1,112 @@
+[//]: # (title: Overview)
+
+Kotlin, through Kotlin Multiplatform, offers two approaches for web development:
+
+* [WebAssembly-based (using the Kotlin/Wasm compiler)](#kotlin-wasm)
+* [JavaScript-based (using the Kotlin/JavaScript compiler)](#kotlin-javascript)
+
+Both approaches let you share code in your web app, but they support different use cases. 
+They also differ in technical aspects such as target browser support.
+
+## Kotlin/Wasm
```

Comment:
The same here, we're mentioning Beta and Experimental things first (and don't mention the stability level of those).
Which I believe is important thing in a high-level overview